config/checkstyle_resources_suppressions.xml

```diff
@@ -167,4 +167,906 @@
   <suppress checks="PackageDeclarationCheck"
             files="[\\/]InputAstTreeStringPrinterTextBlocksEscapesAreOneChar\.java"/>
 
+  <!-- Until https://github.com/checkstyle/checkstyle/issues/11163 -->
+  <suppress checks="FileLength"
+    files="[\\/]InputNonEmptyAtclauseDescription\.java"/>
```

Comment:
Please keep in path name of Check, it will help a lot when we start to resolve such suppression, as we will do them check by check

Reply:
Done